backpack-lint/src/main/java/net/skyscanner/backpack/lint/check/BpkComposeComponentUsageDetector.kt

```diff
@@ -71,6 +71,14 @@ class BpkComposeComponentUsageDetector : Detector(), SourceCodeScanner {
     val webName: String,
     val componentsToReplace: Set<String>,
   ) {
+    BADGE(
```

Comment:
this snuck back in - you might have to rebase the whole thing

Reply:
Hahaha